hw3/02-url.js

```diff
@@ -32,7 +34,28 @@ const server = http.createServer((req, res) => {
   }
 
   // Add your code here
-
+  const result = () => {
+    // search parameters and creat the table and store them in a string
+    const searchParams = new URLSearchParams(url.searchParams);
+    let content = '<table border="1">';
+    searchParams.forEach((value, key) => {
+      content += `<tr><td>${key}</td><td>${value}</td></tr>`;
+    });
+    content += '</table>';
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.write(content);
+  };
+  if (req.url === '/attributes?hello=world&lorem=ipsum') {
+    result();
+  }
+  if (req.url === '/items?first=1&second=2&third=3&fourth=4') {
+    result();
+  }
+  if (
+    req.url === '/characters?spongebob=squarepants&patrick=star&sandy=cheeks'
+  ) {
+    result();
+  }
```

Comment:
The simplicity of your solution to this exercise is great. Specifically, I think your use of the URLSearchParams is great when using forEach to loop through the params.